src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -702,6 +704,70 @@ private IEnumerable<DomainError> BuildUlnValidationFailures(DraftApprenticeshipD
             }
         }
 
+        private IEnumerable<DomainError> BuildFlexibleEmploymentValidationFailures(DraftApprenticeshipDetails draftApprenticeshipDetails)
+        {
+            if (draftApprenticeshipDetails.DeliveryModel != DeliveryModel.PortableFlexiJob)
+            {
+                yield break;
+            }
+
+            foreach (var failure in BuildFlexibleEmploymentPriceValidationFailures(draftApprenticeshipDetails))
+            {
+                yield return failure;
+            }
+
+            foreach (var failure in BuildFlexibleEmploymentDateValidationFailures(draftApprenticeshipDetails))
+            {
+                yield return failure;
+            }
+        }
+
+        private IEnumerable<DomainError> BuildFlexibleEmploymentDateValidationFailures(DraftApprenticeshipDetails draftApprenticeshipDetails)
+        {
+            if (draftApprenticeshipDetails.EmploymentEndDate == null || draftApprenticeshipDetails.TrainingProgramme.EffectiveTo == null)
+            {
+                yield break;
+            }
+
+            if (draftApprenticeshipDetails.EmploymentEndDate.Value < draftApprenticeshipDetails.TrainingProgramme.EffectiveFrom.Value.AddMonths(3))
+            {
+                yield return new DomainError(nameof(draftApprenticeshipDetails.EmploymentPrice), "This date must be at least 3 months later than the planned apprenticeship training start date");
+            }
+
+            if (draftApprenticeshipDetails.EmploymentEndDate.Value > draftApprenticeshipDetails.TrainingProgramme.EffectiveTo)
+            {
+                yield return new DomainError(nameof(draftApprenticeshipDetails.EmploymentPrice), "This date must not be later than the projected apprenticeship training end date");
+            }
+        }
```

Comment:
I think we are using the wrong field here we need to be using the apprenticeship StartDate and EndDate if entered, the date you are referencing here is the Effective Date of the apprenticeship course. ie when the course can be studied